From ac25fde0ab7284737233c0967dda43632e303161 Mon Sep 17 00:00:00 2001
From: Alejandro del Castillo <alejandro.delcastillo@ni.com>
Date: Fri, 22 Feb 2019 12:25:08 -0600
Subject: [PATCH] opkg-key: whitelist gpg.conf during reset

Exclude gpg.conf file from reset, since that file is meant to be used to
customize how gpg works.

Signed-off-by: Alejandro del Castillo <alejandro.delcastillo@ni.com>

Upstream-Status: Submitted
---
 utils/opkg-key | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/opkg-key b/utils/opkg-key
index 8550ea9..2ea9b10 100755
--- a/utils/opkg-key
+++ b/utils/opkg-key
@@ -90,7 +90,7 @@ case "$command" in
         $GPG $*
         ;;
     reset)
-        rm -Rf "$GPGHOMEDIR"
+        find "$GPGHOMEDIR" ! -path "$GPGHOMEDIR/gpg.conf" ! -path "$GPGHOMEDIR" -prune -exec rm -r {} \;
         echo "OK"
         ;;
     populate)
-- 
2.20.1

